nautiluszim/scraper.py

```diff
@@ -303,7 +365,9 @@ def update_metadata(self):
         self.secondary_color = self.secondary_color or secondary_color
 
         # get about content from param, archive or defaults to desc
-        self.about_content = f"<p>{self.description}</p>"
+        self.about_content = (
+            f"<p>{self.description}</p><br><p>{self.long_description}</p>"
```

Comment:
Just use the long_description if it is set and not the description

Reply:
Isn't it better if the user could get a brief understanding of the file by reading the description and then go through the long description for more details. It's just my opinion. I'll change it now, but in case you want me to keep both description and long description, please let me know.

Reply:
Those are primary ZIM Metadata. `Description` and `LongDescription` are exclusive in my understanding (the spec doesnt say anyting and no reader currently uses it) so I'd say no.
The feature you want already sort of exists via the about file that can be included in the archive and would be displayed there

Reply:
Great, thanks for the clarification.